gapic/src/main/com/google/gapid/LoadingScreen.java

```diff
@@ -27,15 +41,30 @@
 import org.eclipse.swt.layout.GridLayout;
 import org.eclipse.swt.widgets.Composite;
 import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Link;
+import org.eclipse.swt.widgets.Listener;
+import org.eclipse.swt.widgets.Menu;
+
+import java.io.File;
 
 /**
  * The loading screen is a minimal view shown while the UI is loading, looking for gapis, etc.
  */
 public class LoadingScreen extends Composite {
+  private final Theme theme;
   private final Label statusLabel;
+  private final Composite optionsContainer;
+  private Label recentIcon;
+  private Link recentLink;
+  private Label helpIcon;
+  private Link helpLink;
```

Comment:
The labels and links can be final.

Reply:
Thanks for all these constructive opinions! Done all the others except this one. Because I initialize my recentLink outside of the constructor. (Cause that part of initialization is huge). Can I keep it or should I still use some tricks to turn it into final? (Like returning recentLink as a variable in the other initializing part)